samples/cellular/modem_shell/sample.yaml

```diff
@@ -8,10 +8,12 @@ tests:
       - nrf9151dk/nrf9151/ns
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161/ns
+      - thingy91x/nrf9151/ns
     platform_allow:
       - nrf9151dk/nrf9151/ns
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161/ns
+      - thingy91x/nrf9151/ns
     tags: ci_build sysbuild
```

Comment:
This CI failure is seen in other modem shell PRs too (no idea yet why SNIPPET="nrf91-modem-trace-uart" doesn't work with thingy91x and why it has got into main in the first place):
`/jenkins_workspace/workspace/latest_sdk-nrf_PR-15685/twister-build/thingy91x_nrf9151_ns/nrf/samples/cellular/modem_shell/sample.cellular.modem_shell.thingy91_modem_uart_trace/b0/../../../../../../../../zephyr/kernel/timeout.c:80: undefined reference to 'sys_clock_elapsed'`

Reply:
I've resolved this and will submit the fix to use SNIPPETS correctly with sysbuild.
PR where this is included is #15225.